tests/gumball-machine-test.ts

```diff
@@ -445,40 +498,23 @@ describe("gumball-machine", () => {
       "Failed to remove lamports from gumball machine acct"
     );
 
-    const approxExpectedAuthorityAcctBalance = originalAuthorityAcctBalance + originalGumballMachineAcctBalance
+    const expectedAuthorityAcctBalance = originalAuthorityAcctBalance + originalGumballMachineAcctBalance
     assert(
-      Math.abs(approxExpectedAuthorityAcctBalance - await connection.getBalance(authorityKeypair.publicKey)) < 10000,
+      expectedAuthorityAcctBalance === await connection.getBalance(authorityKeypair.publicKey),
       "Failed to transfer correct balance to authority"
     );
   }
 
-  describe("Testing gumball_machine", async () => {
-    const baseGumballMachineHeader: InitGumballMachineProps = {
-      urlBase: Buffer.from("https://arweave.net/Rmg4pcIv-0FQ7M7X838p2r592Q4NU63Fj7o7XsvBHEEl"),
-      nameBase: Buffer.from("zfgfsxrwieciemyavrpkuqehkmhqmnim"),
-      symbol: Buffer.from("pehjjqmrjpfcnttlierdqkxjueqjqjsf"), 
-      sellerFeeBasisPoints: 100,
-      isMutable: true,
-      retainAuthority: true,
-      price: new BN(10),
-      goLiveDate: new BN(1234.0),
-      mint: wrappedSolPubKey,
-      botWallet: Keypair.generate().publicKey,
-      authority: payer.publicKey,
-      collectionKey: payer.publicKey,
-      creatorAddress: payer.publicKey,
-      extensionLen: new BN(28),
-      maxMintSize: new BN(10),
-      maxItems: new BN(250)
-    };
-
-    let gumballMachineAcctKeypair;
-    let merkleRollKeypair;
-    let noncePDAKey;
-    const GUMBALL_MACHINE_ACCT_SIZE = 8352;
-    const GUMBALL_MACHINE_ACCT_HEADER_SIZE = 352;
+  describe("Testing gumball-machine", async () => {
+    let baseGumballMachineHeader: InitGumballMachineProps;
+    let creatorAddress: Keypair;
+    let gumballMachineAcctKeypair: Keypair;
+    let merkleRollKeypair: Keypair;
+    let noncePDAKey: PublicKey;
+    const GUMBALL_MACHINE_ACCT_HEADER_SIZE = 384;
```

Comment:
this should be exposed as a header in `../sdk/gumball-machine/index.tsx` or at least at the top of this test